src/state/comments.ts

```diff
@@ -1,330 +1,291 @@
 import type { Annotation } from '../utils/annotation';
 import * as actions from '../actions/comments';
-
-type Partial<T> = {
-    [P in keyof T]?: T[P];
-};
+import { Partial, update } from './utils';
```

Comment:
**Suggestion:** Isn't `Partial` built into TypeScript? You should rename it if it's different to the build-in type or just use what TypeScript has to offer. May cause confusion.

https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Hmm. Yeah, this was historical. I've removed, seems to be working fine so far.